src/PatientChecking/PatientChecking/Services/ServiceModels/Address.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+#nullable disable
+
+namespace PatientChecking.Services.ServiceModels
+{
+    public partial class Address
+    {
+        public int AddressId { get; set; }
+        public int TypeAddress { get; set; }
```

Comment:
Should create an enum for TypeAddress. Marked it as TODO when implement the ticket.